dspace-api/src/main/java/org/dspace/app/requestitem/CollectionAdministratorsRequestItemStrategy.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.app.requestitem;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.dspace.content.Collection;
+import org.dspace.content.Item;
+import org.dspace.core.Context;
+import org.dspace.eperson.EPerson;
+
+/**
+ * Derive request recipients from groups of all collections which hold an Item.
+ * The list will include all members of the administrators group.  If the
+ * resulting list is empty, delegates to {@link RequestItemHelpdeskStrategy}.
+ *
+ * @author Mark H. Wood <mwood@iupui.edu>
+ */
+public class CollectionAdministratorsRequestItemStrategy
+        implements RequestItemAuthorExtractor {
+    @Override
+    public List<RequestItemAuthor> getRequestItemAuthor(Context context,
+            Item item)
+            throws SQLException {
+        List<RequestItemAuthor> recipients = new ArrayList<>();
+        for (Collection collection : item.getCollections()) {
```

Comment:
this should be limited to owning collection otherwise there is a risk of misuse from mapping collections admin that could use the request a copy feature to authorize their self to access restricted bitstreams for item that they have mapped in their collection

Reply:
Agreed.  Done.